upgrade/Stage_0_Prerequisites.md

```diff
@@ -215,16 +215,18 @@ after a break, always be sure that a typescript is running before proceeding.
    git push
    ```
 
-## Stage 0.3 - Update NCN CFS configuration
+## Stage 0.3 - Customize the new NCN image and update NCN personalization configurations
 
 There are two possible scenarios. Follow the procedure for the scenario that is applicable to the upgrade being performed.
 
 - [Standard upgrade](#standard-upgrade) - Upgrading CSM on a system that has products installed other than CSM.
-- [CSM-only system upgrade](#csm-only-system-upgrade) - Upgrading CSM on a system that has CSM installed and **no other products installed**.
+- [CSM-only system upgrade](#csm-only-system-upgrade) - Upgrading CSM only on a CSM-only system **no other products installed or being upgraded**.
+
+**NOTE:** It will not be possible to rebuild NCNs on the current, pre-upgraded CSM version after performing these steps. Rebuilding NCNs will become the same thing as upgrading them.
```

Comment:
Isn't this note only true for the standard upgrade path?

Reply:
> I made one linting commit to this PR, and depending on the answer to my question, an additional change will be needed. That will need to also get added to the backport.

I undid the backport, will re-backport after this is settled.

Reply:
addressed